app/models/user.rb

```diff
@@ -3,7 +3,10 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: :author_id, dependent: :destroy
   has_many :likes, foreign_key: :author_id, dependent: :destroy
 
-  def most_recent_three_posts
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  def three_most_recent_posts
```

Comment:
- [  ] Good job you did 👏 . However, kindly add unit specs for `validations` as specified in the requirenet.

### Requirements:
![image](https://github.com/mohamedSabry0/blog_hub/assets/50990925/afe70681-1399-4fb7-9b3d-61e5e33053ad)

### Sample code to add:
```
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```

- Kindly do this for all other models `Post` `Comment` and `Like`